.github/workflows/commands.yml

```diff
@@ -5,7 +5,13 @@ on:
 
 jobs:
   retest:
-    if: github.repository == 'envoyproxy/envoy'
+    if: |
+      ${{
+         github.event.issue.pull_request
+         && github.repository == 'envoyproxy/envoy'
+         && github.actor != 'repokitteh-read-only[bot]'
+         && github.actor != 'dependabot[bot]'
```

Comment:
i think these conditions should work correctly - but not 100% - would be good to exclude all bots

Reply:
because AND is use here, so seems this new condition is more strict?